### PR TITLE
✨ Add conditional-row to Lovelace entities card

### PR DESCRIPTION
**Pull request by iantrich**
**Description:**
https://github.com/home-assistant/home-assistant-polymer/pull/4569

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_lovelace/entities.markdown [hunk 1]
```diff
@@ -222,6 +222,42 @@ hide_if_unavailable:
...
+  required: true
+  description: List of entity IDs and matching states.
+  type: list
+  keys:
+    entity:
+      required: true
+      description: HA entity ID.
+      type: string
+    state:
+      required: false
```
Comment (bramkragten) [inline source/_lovelace/entities.markdown:244]: I would place the asterisk here

File: source/_lovelace/entities.markdown [hunk 2]
```diff
@@ -222,6 +222,42 @@ hide_if_unavailable:
...
+  type: string
+conditions:
+  required: true
+  description: List of entity IDs and matching states.
+  type: list
+  keys:
+    entity:
+      required: true
+      description: HA entity ID.
+      type: string
```
Comment (bramkragten) [inline source/_lovelace/entities.markdown]: Aren't entities and state lists like with an `entity-filter`?

File: source/_lovelace/entities.markdown [hunk 3]
```diff
@@ -222,6 +222,42 @@ hide_if_unavailable:
...
+    state:
+      required: false
+      description: Entity state is equal to this value.*
```
Comment (klaasnicolaas) [inline source/_lovelace/entities.markdown:245]: You can't use a ⭐️ to indicate that something is required, Markdown doesn't like that.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_lovelace/entities.markdown (hunk 3 above)
Comment (iantrich) [reply source/_lovelace/entities.markdown:245]: The `*` just highlights the additional information

Comment (iantrich) [reply source/_lovelace/entities.markdown:245]: Ah, were you referring to @bramkragten suggestion to move the *, as that did indeed break the parser?

**Review by frenck — APPROVED** [review]

**Pull request merged**